specification/k3po.control/SPEC.md

```diff
@@ -114,8 +92,7 @@ N/A
 ##### ABORT command
 
 ###### Description
-This command will abort the current execution. It may be sent any time after a PREPARE command. When the execution of the scripts is aborted,
-a FINISHED event may be returned with the script executed so far.
+The **ABORT** command will abort the currently running script execution.
 
```

Comment:
ABORT will stop the execution of all scripts in the current test. No matter if a script is running or blocked waiting for an event.